### PR TITLE
columnedMultiLineOutput - copy all values option

### PR DESCRIPTION
**Pull request by Mike-3**

### CODE REVIEW

File: lib/widgets/utils/common_widget_utils.dart [hunk 1]
```diff
@@ -37,7 +37,7 @@ defaultFontSize() {
 }
 
 List<Widget> columnedMultiLineOutput(BuildContext context, List<List<dynamic>> data,
-    {List<int> flexValues = const [], int copyColumn, hasHeader: false, List<Function> tappables}) {
+    {List<int> flexValues = const [], int copyColumn, bool hasHeader: false, bool copyValueColumn: false, List<Function> tappables}) {
```
Comment (S-Man42) [inline lib/widgets/utils/common_widget_utils.dart:40]: I don't like the name :D Maybe better: copyAll

File: lib/widgets/utils/common_widget_utils.dart [hunk 2]
```diff
@@ -62,6 +62,12 @@ List<Widget> columnedMultiLineOutput(BuildContext context, List<List<dynamic>> d
 
     if (copyColumn == null) copyColumn = rowData.length - 1;
     var copyText = rowData[copyColumn].toString();
+    if ((isFirst && hasHeader) & copyValueColumn) {
```
Comment (S-Man42) [inline lib/widgets/utils/common_widget_utils.dart:65]: Is te & operator correct here (and below). Shouldn't it be: &&

Comment (Mike-3) [reply lib/widgets/utils/common_widget_utils.dart:65]: Why not ?
(isFirst && hasHeader) is original, 
I can change this to (isFirst & hasHeader & copyValueColumn)

File: lib/widgets/utils/common_widget_utils.dart (hunk 1 above)
Comment (Mike-3) [reply lib/widgets/utils/common_widget_utils.dart:40]: I also had it first. But only the Value column is copied. If everything has to be copied, it doesn't fit. 
copyValues ?

File: lib/widgets/utils/common_widget_utils.dart (hunk 2 above)
Comment (S-Man42) [reply lib/widgets/utils/common_widget_utils.dart:65]: To be honest, I am not familiar with the bitwise AND. Does it really do the same like &&? 

I'd write 

`isFirst && hasHeader && copyValueColumn`

File: lib/widgets/utils/common_widget_utils.dart (hunk 1 above)
Comment (S-Man42) [reply lib/widgets/utils/common_widget_utils.dart:40]: copyValues is not better... copyAllCopyValues? :D Hm, I believe "copyAll" is not correct, sure - but it's close enough...

**Pull request merged**